datalab/datalab.sh

```diff
@@ -18,6 +18,12 @@
 
 set -exo pipefail
 
+# Exit immediatedly if this is not a master node, because
```

Comment:
It is more readable to put this check in main().

Reply:
Done